### ISSUE TITLE
Tflite Custom model export issue

### ISSUE THREAD

**Issue by rahuja23** [ultralytics/yolov5#5190]
When locally trying to convert ".pt" file to tflite using export.py, I am getting an error:
 `export failure: name 'SPPF' is not defined`
 on macbook. To debug this I import SPPF module from models.common.py but then I get a new error:
`TensorFlow saved_model: export failure: name 'TFSPPF' is not defined`

In previous version of yolov5 repo when I am exporting a custom trained model into tflite and then loading it , I am getting the following error:
`Model provided has model identifier '`

Any helps or suggestions what can be done?

**Comment by glenn-jocher** [comment]
@rahuja23 your code is out of date, update your code.

**Comment by raccoondev85** [comment]
@glenn-jocher I donwloaded release source v6.0.zip and exported my custom model,, then i got similar error. it was okay with previous version though...

(base) root@3f6378e93877:/yolov5-master_03# python export.py --weights ./export_tflite/yolov5s/best.pt --include tflite
export: data=data/coco128.yaml, weights=./export_tflite/yolov5s/best.pt, imgsz=[640, 640], batch_size=1, device=cpu, half=False, inplace=False, train=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=13, topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['tflite']
YOLOv5 🚀 2021-10-12 torch 1.9.0+cu102 CPU

Fusing layers... 
Model Summary: 213 layers, 7020913 parameters, 0 gradients, 15.8 GFLOPs

PyTorch: starting from export_tflite/yolov5s/best.pt (56.8 MB)
2021-11-02 10:28:24.178321: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0

TensorFlow saved_model: starting export with tensorflow 2.4.1...

                 from  n    params  module                                  arguments                     
2021-11-02 10:28:25.225959: I tensorflow/compiler/jit/xla_cpu_device.cc:41] Not creating XLA devices, tf_xla_enable_xla_devices not set
2021-11-02 10:28:25.227028: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcuda.so.1
2021-11-02 10:28:25.254770: E tensorflow/stream_executor/cuda/cuda_driver.cc:328] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2021-11-02 10:28:25.254825: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:169] retrieving CUDA diagnostic information for host: 3f6378e93877
2021-11-02 10:28:25.254833: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:176] hostname: 3f6378e93877
2021-11-02 10:28:25.255003: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:200] libcuda reported version is: 470.57.2
2021-11-02 10:28:25.255025: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:204] kernel reported version is: 470.57.2
2021-11-02 10:28:25.255033: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:310] kernel version seems to match DSO: 470.57.2
2021-11-02 10:28:25.255367: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-11-02 10:28:25.256993: I tensorflow/compiler/jit/xla_gpu_device.cc:99] Not creating XLA devices, tf_xla_enable_xla_devices not set
  0                -1  1      3520  models.common.Conv                      [3, 32, 6, 2, 2]              
  1                -1  1     18560  models.common.Conv                      [32, 64, 3, 2]                
  2                -1  1     18816  models.common.C3                        [64, 64, 1]                   
  3                -1  1     73984  models.common.Conv                      [64, 128, 3, 2]               
  4                -1  1    115712  models.common.C3                        [128, 128, 2]                 
  5                -1  1    295424  models.common.Conv                      [128, 256, 3, 2]              
  6                -1  1    625152  models.common.C3                        [256, 256, 3]                 
  7                -1  1   1180672  models.common.Conv                      [256, 512, 3, 2]              
  8                -1  1   1182720  models.common.C3                        [512, 512, 1]                 

TensorFlow saved_model: export failure: name 'SPPF' is not defined

TensorFlow Lite: starting export with tensorflow 2.4.1...

TensorFlow Lite: export failure: 'NoneType' object has no attribute 'call'

**Comment by raccoondev85** [comment]
Just pulled the recent repo code exported tflite successfully, 
I was planning to maintain my source code with the official releases but it'll be difficult to follow after all.
hope to release mini patched version :)

**Comment by glenn-jocher** [comment]
@raccoondev85 your code is out of date. Update your code to master.
